src/datasources/fault-ds/mapping/ValueMapping.js

```diff
@@ -1,34 +1,10 @@
 
 export class ValueMapping {
```

Comment:
Is it worth even keeping this class, if all it does is return the given `value`?